### PR TITLE
Cl tool

### PR DESCRIPTION
**Pull request by CarolineMorton**
This PR aims to make this a simple command line tool that can be called within job runner and locally. 

It looks in the yaml for 2 keys 
- `inputs` - input files
- `config` - this is the table configuration. 

The `test_json_1.json` is a json version of the yaml below:
```yaml
inputs: ["tests/test_data/test_data.csv"]
config:
  simple_2_way_tabs:
    tab_type: 2-way
    variables: [sex, ageband, copd]
  death_2_way_tab:
    tab_type: target-2-way
    variables: [sex, ageband, copd]
    target: death
  grouped_by_sex:
    tab_type: groupby-2-way
    variables: [ageband, copd, death]
    groupby: sex
```

Safetab can be called from the command line with
`python entrypoint.py "tests/test_json_configs/test_json_1.json" for that example.

### CODE REVIEW

File: entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    parser.add_argument("--version", action='version', version="safetab 0.0.1")
+    
+    # configurations
+    parser.add_argument("config", type=str, help="The yaml passed in as a json obj")
+
+    # parse args
+    args = parser.parse_args()
+    
+    kwargs = vars(args)
+    yaml_str = kwargs.pop("config")
```
Comment (bloodearnest) [inline entrypoint.py:32]: This is not yaml, it's json, and it's not a string of json, it's a path to json data, so maybe `json_path` is a better variable name?

File: safetab/find_save_tools.py [hunk 2]
```diff
@@ -33,6 +33,7 @@ def make_folders(table_config_json, path=None):
         for folder_name, table_dets in folder_names.items():
             os.mkdir(folder_name)
     else:
+        os.mkdir(path)
```
Comment (bloodearnest) [inline safetab/find_save_tools.py:36]: Maybe add `(exist_ok=True)` in case the dir already exist?

**Review by bloodearnest — APPROVED** [review]
LGTM, left a few minor non-blocking suggestions.

I update the issue discussing the action calling convention this is based on: https://github.com/opensafely-core/job-runner/issues/124#issuecomment-811342905

File: entrypoint.py (hunk 1 above)
Comment (CarolineMorton) [reply entrypoint.py:32]: agree - changed

File: safetab/find_save_tools.py (hunk 2 above)
Comment (CarolineMorton) [reply safetab/find_save_tools.py]: done - thanks for the tip

**Pull request merged**